redis/connection.py

```diff
@@ -1527,7 +1670,7 @@ def reset(self):
 
         # Keep a list of actual connection instances so that we can
         # disconnect them later.
-        self._connections = []
+        self._connections: List[Connection] = []
```

Comment:
if you're using `__future__` annotations, then you don't need to use `List[Connection]`; built-in list is ok: `list[Connection]` which removes an import from typing

Reply:
That's ok, I'm just worried about the code conflicting until then. Anything I fix, anyway. I'll first do a commit using `typing.List` and then do another one by fixing it.

Reply:
I am afraid that `list[Connection]` is just a python3.10 thing

Reply:
it is also available via future.annotations

Reply:
that's interesting... thanks for letting me know